docs/sources/panels/visualizations/_index.md

```diff
@@ -1,10 +1,75 @@
 +++
-title = "Visualization options"
+title = "Visualizations"
 weight = 420
 +++
 
-# Visualization options
+# Visualizations
```

Comment:
maybe?
```suggestion
# Visualization panels
```

Do we want to make it more clear that "panels" == "visualizations"?  And adding new "panel plugins" adds more visualization options?